src/contractIntegration/interaction/IInteractionConfig.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable  @typescript-eslint/no-explicit-any */
```

Comment:
@HectorGerardoZV ABI could be typed using  [web3-utils](https://www.npmjs.com/package/web3-utils)